recipes/openfe/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openfe" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenFreeEnergy/openfe/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 03b6531ed7c8b03f09294c9792ce465633c1d2e6aa60d0e34344a14b3dc56b96
+
+build:
+  noarch: python
```

Comment:
You'll need to add the entry points here as well.

Ref:

* https://github.com/OpenFreeEnergy/openfe/blob/03cf8a1ca3db5af90a6220742dc2d105173cc0e6/setup.cfg#L41-L43
* https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points